arrays/arrays.c

```diff
@@ -107,10 +107,15 @@ void arr_append(Array *arr, char *element) {
 
     // Resize the array if the number of elements is over capacity
     // or throw an error if resize isn't implemented yet.
+    if (arr->count == arr->capacity) {
+        printf("Not enough memory allocated to append to this array");
```

Comment:
not suuper important since you're implementing `resize` in a bit, but:
if there's no capacity and you print the error, you also want it to break out - otherwise the function keeps going, and line 115 adds more elements to an array that doesn't have any more capacity